linux_os/guide/system/accounts/accounts-restrictions/password_storage/no_empty_passwords/ansible/ubuntu2004.yml

```diff
@@ -0,0 +1,11 @@
+# platform = multi_platform_ubuntu
```

Comment:
It is usually better to use the common name (shared.yml) for remediation and control its restrictions with headers. It make them easier to be adopted by other products, if necessary.

Reply:
Thanks